neon_utils/configuration_utils.py

```diff
@@ -928,3 +928,11 @@ def create_config_from_setup_params(path=None) -> NGIConfig:
     # TODO: Use XDG here DM
     local_conf.write_changes()
     return local_conf
+
+
+def get_default_local_config(path=None) -> dict:
```

Comment:
Why use this vs `get_neon_local_config`? https://github.com/NeonGeckoCom/neon-skill-utils/blob/4b7b6d86b9dfd0e96fa6ab9f8fcdbe8b4bf26205/neon_utils/configuration_utils.py#L753

Reply:
> Why use this vs `get_neon_local_config`?
> 
> https://github.com/NeonGeckoCom/neon-skill-utils/blob/4b7b6d86b9dfd0e96fa6ab9f8fcdbe8b4bf26205/neon_utils/configuration_utils.py#L753

Surely we can, just refer to the advice you gave me on Slack